llama-index-core/llama_index/core/indices/struct_store/sql_query.py

```diff
@@ -423,6 +427,8 @@ def _query(self, query_bundle: QueryBundle) -> Response:
                 nodes=retrieved_nodes,
             )
             cast(Dict, response.metadata).update(metadata)
+            if self._streaming:
+                return cast(StreamingResponse, response)
```

Comment:
This doesn't actually help 😅 The function signature is `-> Response`

Instead, it should probably be `-> RESPONSE_TYPE`, and any functions using `_query()` should deal with that output type

(this would have been caught already if mypy was working, I actually have a WIP PR for getting mypy working on core again, and I think I already fixed this in that branch)

Reply:
Ah of course

Reply:
Updated to RESPONSE_TYPE